conf/modules.config

```
@@ -132,15 +132,15 @@ process {
             path: { "${params.outdir}/protein_annotation/interproscan/" },
             mode: params.publish_dir_mode,
             enabled: params.run_protein_annotation,
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
         ]
         ext.args   = [
             "--applications ${params.protein_annotation_interproscan_applications}",
             params.protein_annotation_interproscan_enableprecalc ? '' : '--disable-precalc',
             '--disable-residue-annot',
             '--enable-tsv-residue-annot',
-            "--formats tsv"
-        ].join(' ').trim() // Warning: Do not disable the flags "--enable-tsv-residue-annot" and "--formats tsv"! This would cause a run failure because the format of the resulting files would no longer be adequate for parsing by AMPcombi2.
+            "--formats tsv",
+        ].join(' ').trim()
```

```suggestion
        ].join(' ').trim() // Warning: Do not disable the flags "--enable-tsv-residue-annot" and "--formats tsv"! This would cause a run failure because the format of the resulting files would no longer be adequate for parsing by AMPcombi2.
```

I would leave this comment as is.